package.json

```diff
@@ -18,6 +18,7 @@
     "@angular/compiler": "^15.1.0",
     "@angular/core": "^15.1.0",
     "@angular/forms": "^15.1.0",
+    "@angular/material": "^17.2.2",
```

Comment:
anugular material we are not using - we are using primeng

Reply:
I am using the angular material for the dropdown in association feature.